app/templates/components/event-invoice/invoice-summary.hbs

```diff
@@ -0,0 +1,35 @@
+<div class="ui segments">
+  <div class="ui green inverted segment center aligned">
+    <div class="ui inverted mini statistic horizontal">
+      <div class="value">
+        {{t 'Paid'}}
+      </div>
+      <div class="label">
+        {{t 'Your invoice payment completed successfully.'}}
+        <br>
+        {{t 'Find all the details related to your invoice below.'}}
+      </div>
+    </div>
+  </div>
+  <div class="ui secondary segment">
+    <h3 class="weight-400">{{t 'Invoice Details'}}</h3>
+  </div>
+  <table class="ui very basic tablet stackable table order-summary center aligned">
```

Comment:
unstackable

Reply:
@kushthedude if I go with unstackable then I'll have to take care that the table data is not getting out of the table in mobile view.
Moreover, the order page are implemented on stackable concept. I think it'll prevent overflow even if we add few more info to table later

Reply:
@shreyanshdwivedi , There will be loose headers all over which may be much difficult to understand for the user ?

Reply:
Let me add a screenshot for mobile view. I think it is ok
@uds5501 @mrsaicharan1 ?
![Screenshot_2019-08-03 Paid Event Invoice -b0929ea6-1b8f-4dea-8075-4f3b437d7325 Open Event](https://user-images.githubusercontent.com/25428397/62392550-5736d580-b585-11e9-8687-d1ca91519b96.png)